tests/all/test_main.py

```diff
@@ -162,3 +164,36 @@ def test_verbose_flag(flag_val, expected_value):
         args.append(flag_val)
     parsed = ert_parser(None, args)
     assert parsed.verbose == expected_value
+
+
+def test_version_valid_Version(capsys):
+
+    try:
+        ert_parser(None, ["--version"])
+    except SystemExit as e:
+        assert e.code == 0
+
+    if sys.version_info.major < 3:
```

Comment:
I'm not quite sure why the test returns here? If this isn't supposed to be tested in python 2, then you should use a `@skip` decorator on the test. See https://github.com/equinor/ert/blob/master/tests/all/plugins/test_plugin_context.py#L28

Reply:
It's just the `capsys` decorator that has a bug in python 2, but the pytest developers have chosen not to fix it :man_shrugging: So the test of the exit code works, it's just that it does not capture stderr.